core/deployment/src/main/java/io/quarkus/deployment/builditem/nativeimage/NativeImageResourceBuildItem.java

```diff
@@ -7,7 +7,11 @@
 import io.quarkus.builder.item.MultiBuildItem;
 
 /**
- * A build item that indicates that a static resource should be included in the native image
+ * A build item that indicates that a static resource should be included in the native image.
+ * <p>
+ * A static resource is a file that is not processed by the build steps, but is included in the native image as-is.
```

Comment:
I thought we should mention that the resource is a `/`-separated path name (with the same semantics as `java.lang.ClassLoader.getResource(String)`?).

Reply:
Ack, please take another look.